Makefile

```diff
@@ -80,9 +80,11 @@ help:
 	@echo '  lint-install - install linter'
 	@echo '  check-deps   - check docs/LICENSE_OF_DEPENDENCIES.md'
 	@echo '  clean        - delete build artifacts'
+	@echo '  package      - build all supported packages, override include_packages to only build a subset'
+	@echo '                 e.g.: make package include_packages="amd64.deb"'
 	@echo ''
-	@echo 'Package Targets:'
-	@$(foreach dist,$(dists),echo "  $(dist)";)
+	@echo 'Possible values for include_packages variable'
```

Comment:
This returns strings that describe the OS, architecture, and format. It used to list the filenames of the packages. I think we should try to do that again because it's more descriptive.

Reply:
My concern here is that part of the package name is calculated in the makefile based on the packaging type including the next version and the commit sha. Then the user will have to figure out the version and commit sha manually before they can use `include_package`. For example the user would have to type this  `make package include_packages="telegraf_1.20.0~aa72552f-0_amd64.deb"`. Maybe a solution would be to add the possible package names for deb,rpm,tar,zip in the help output but then not require them for include_package. Something like:

```
Resulting package name format (where arch will be the arch of the package):
	telegraf_1.20.0~aa72552f-0_arch.deb
	telegraf-1.20.0~aa72552f-0.arch.rpm
	telegraf-1.20.0~aa72552f_arch.tar.gz
	telegraf-1.20.0~aa72552f_arch.zip
```